Bio.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+  <head>
+  <meta charset="utf-8">
+  <title>Lebron James Portfolio</title>
+  <link rel="stylesheet" type="text/css" href="css/style.css">
+  <link href="https://fonts.googleapis.com/css?family=Rock+Salt" rel="stylesheet">
+  </head>
+  
+  <body>
+
+    <nav>
+      <strong>Bio</strong>
+      |
+      <a href="Experience.html">Experience</a>
+     </nav>
+
+    <h1>Who am I?</h1> 
+
+    <hr>
+
+    <img class="witnesses" src="images/WeAreAllWitnesses.jpg">
```

Comment:
Although not required, the best practice is to include the `alt` attribute which allows for accessibility to the visually impaired and is what appears if the resource fails to load.